rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -472,6 +473,31 @@ def send_blueprint(
     )
 
 
+def send_recording(recording: Recording, stream: RecordingStream | None = None) -> None:
```

Comment:
Do we need this Python function at all? Can this logic be done in the Rust implementation instead? @jleibs has recently implemented a bunch of APIs for the Python SDK that don't have a single line of Python in them (AFAIK?), and I would very much like to keep that trend...

Reply:
I looked into it, but it doesn’t seem super straightforward in this case. There are a few things that make it a bit tricky, e.g. `RecordingStream` wrapping `PyRecordingStream`

Reply:
Yes. This can only be done correctly once everything else is done correctly. By "correctly", I mean according to this issue:
- https://github.com/rerun-io/rerun/issues/9187